### PR TITLE
Finished Aphrodite restyle of safebrowsing.js. Fixes issue #9025

### PR DESCRIPTION
**Pull request by jasonrsadler**
## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request [as needed](https://github.com/brave/browser-laptop/wiki/Pull-request-process).
- [x] Request a security/privacy review [as needed](https://github.com/brave/handbook/blob/master/development/security.md#how-to-request-a-security-review). (Ask a Brave employee to help if you cannot access this document.)

## Reviewer Checklist:

- [ ] Request a security/privacy review [as needed](https://github.com/brave/handbook/blob/master/development/security.md#how-to-request-a-security-review) if one was not already requested.

Tests

- [ ] Adequate test coverage exists to prevent regressions
- [ ] Tests should be independent and work correctly when run individually or as a suite [ref](https://github.com/brave/browser-laptop/wiki/Code-Guidelines#test-dependencies)
- [ ] New files have MPL2 license header

Finishes restyle to Aphrodite. Eliminates dependency to 'less' files. 
Requesting Review @luixxiul @bsclifton 
Requesting Security Review @diracdeltas

### CODE REVIEW

**Comment by jasonrsadler** [comment]
Cleaning up open PRs

File: js/about/safebrowsing.js [hunk 1]
```diff
@@ -7,50 +7,51 @@ const BrowserButton = require('../../app/renderer/components/common/browserButto
...
     super(props)
     this.state = {
       advanced: false
     }
+    let headDiv = document.querySelector('#appContainer')
+    headDiv.className = css(styles.appContainer)
   }
 
-  onAdvancedToggle () {
+  onAdvancedToggle = () => {
```
[the remark this answers is not in the document]
Comment (jasonrsadler) [reply js/about/safebrowsing.js]: To prevent having to call .bind from each usage.

File: app/renderer/components/styles/globalSelectors.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+const globalStyles = require('./global')
+
+const GLOBALS = '__GLOBAL_STYLES__'
+
+const globalExtension = {
+  selectorHandler: (selector, baseSelector, generateSubtreeStyles) =>
+    (baseSelector.includes(GLOBALS) ? generateSubtreeStyles(selector) : null)
+}
+
+export const childSelector = (selector) => {
```
Comment (cezaraugusto) [inline app/renderer/components/styles/globalSelectors.js]: is this being used somewhere?

[the remark this answers is not in the document]
Comment (jasonrsadler) [reply app/renderer/components/styles/globalSelectors.js]: No. that's a leftover with no cleanup. Removing childSelector

File: app/renderer/components/styles/globalSelectors.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+const styles = extended.StyleSheet.create({
+  [GLOBALS]: {
+    '*': {
+      color: globalStyles.color.commonTextColor,
+      fontFamily: 'Helvetica Neue, Arial, sans-serif',
+      fontWeight: 400,
+      margin: 0,
+      padding: 0
+    },
```
[the remark this answers is not in the document]
Comment (jasonrsadler) [reply app/renderer/components/styles/globalSelectors.js]: Yes, I saw that on the styling guidelines after the fact. Done.

File: app/renderer/components/styles/globalSelectors.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  return `${selector}`
+}
+
+const extended = StyleSheet.extend([globalExtension])
+
+const styles = extended.StyleSheet.create({
+  [GLOBALS]: {
+    '*': {
+      color: globalStyles.color.commonTextColor,
+      fontFamily: 'Helvetica Neue, Arial, sans-serif',
```
[the remark this answers is not in the document]
Comment (jasonrsadler) [reply app/renderer/components/styles/globalSelectors.js]: Done.

File: app/renderer/components/styles/globalSelectors.js [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+    // when font-size is greater than 20px and less than 30px
+    '@typographyDisplayMediumSpacing': '-0.2px',
+
+    // used for body / flowing text
+    '@typography-body': {
+      fontFamily: 'Muli, -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen, Ubuntu, Cantarell, Open Sans, Helvetica Neue, sans-serif'
+    }
+  }
+})
+export default extended.css(styles[GLOBALS])
```
[the remark this answers is not in the document]
Comment (jasonrsadler) [reply app/renderer/components/styles/globalSelectors.js]: Agree. Done.

File: js/about/safebrowsing.js [hunk 6]
```diff
@@ -7,50 +7,51 @@ const BrowserButton = require('../../app/renderer/components/common/browserButto
...
+const globalStyles = require('../../app/renderer/components/styles/global')
+require('../../app/renderer/components/styles/globalSelectors')
 
 class SafebrowsingPage extends React.Component {
   constructor (props) {
     super(props)
     this.state = {
       advanced: false
     }
+    let headDiv = document.querySelector('#appContainer')
```
[the remark this answers is not in the document]
Comment (jasonrsadler) [reply js/about/safebrowsing.js]: Agree. Done.

File: app/renderer/components/styles/globalSelectors.js [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+    '*': {
+      color: globalStyles.color.commonTextColor,
+      fontFamily: 'Helvetica Neue, Arial, sans-serif',
+      fontWeight: 400,
+      margin: 0,
+      padding: 0
+    },
+    'html, body, #appContainer, #appContainer > div': {
+      height: '100%'
+    },
```
[the remark this answers is not in the document]
Comment (jasonrsadler) [reply app/renderer/components/styles/globalSelectors.js]: This one I might need your help understanding.

**Comment by jasonrsadler** [comment]
@cezaraugusto Thank you. Updated with corrections.

**Comment by bsclifton** [comment]
master e278ba62b8fcb92f26f18de15a2950d14478aac3
0.23.x 688f8f99bbbd4fd4c60e44fedbd66766a93f419e

**Comment by bsclifton** [comment]
@jasonrsadler @cezaraugusto I uplifted this to 0.23.x to avoid getting ourselves into a nasty merge conflict type situation (like we had in the past when cherry picking commits to older releases)

**Comment by cezaraugusto** [comment]
Labeled as `no-qa-needed` as no functionality was added and no design was changed. Thanks all

**Pull request merged**